src/Microsoft.Health.Operations.Functions/Management/PurgeOrchestrationInstanceHistory.cs

```diff
@@ -42,6 +45,7 @@ public PurgeOrchestrationInstanceHistory(IOptions<PurgeHistoryOptions> options)
         PurgeHistoryOptions value = EnsureArg.IsNotNull(options?.Value, nameof(options));
```

Comment:
Should we just save the `PurgeHistoryOptions` as the member here?

Reply:
We could, but then we also do some validations in the below method to check for nulls

Reply:
We can check for nulls while just keeping the options, right?

Reply:
Not sure If I explained what I meant. I updated the code to have options as  member variable but need to move null checks to Run method.